apps/healthlocker/web/controllers/button_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Healthlocker.ButtonController do
+  use Healthlocker.Web, :controller
+
+  def index(conn, _params) do
+    # for now this only uses one button, so the same image is loaded on error
+    # in the future, the case would try read a file that's put in as a param,
+    # and if it didn't exist it would load the default.
+    button =
+      case File.read("web/static/assets/images/epjs_button.png") do # won't be a dog
+        {:ok, binary} ->
+          Base.encode64(binary)
+        {:error, _} ->
+          binary = File.read!("web/static/assets/images/epjs_button.png")
```

Comment:
I think this path is broken. maybe try with ```apps/healthlocker/web/static/assets/images/epjs_button.png```
I'm not sure what is the role of this controller. The index.html only display the image, do we still need this?